contents/handbook/company/employee-expectations.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Expectations for PostHog employees
+sidebar: Handbook
+showTitle: true
+---
+
+## Introduction
+
+People who work at PostHog have come from all sorts of different backgrounds - large companies, small companies, agencies, single-founder startups. The modes of working at these places necessarily differ from each other and from PostHog. However, when you join PostHog, we expect you to work in some fairly specific ways. And being the transparent company that we are, we want to let you know about those expectations, because you can only meet expectations when you are aware of them.
+
+Generally, our [values](/handbook/values) are a great place to start, as well as our doc on [culture](/handbook/company/culture). Here are a few specific ways to apply those values:
+
+## Ask for help, but only after you've tried first
+
+If you've just joined us, there's a lot you probably don't know. That's okay! However, we _do_ expect that you try to help yourself. Here's a framework to use as a guide:
+
+- If it's one of our own products that we build and sell
+  - We expect you to [read our docs](/docs)
+  - We expect you to search through GitHub or Slack
+  - If it's in beta, it might be buggy or missing features - definitely request/report them, but also figure out how to do your job with plan B and plan C if it won't be fixed quickly -- this is what we generally call grit
+- If it's an external product
+  - We expect you to read the docs
+  - If it takes you more than 10 mins to figure it out, then ask someone internally.
+- If it's code
+  - Spend at least an hour on trying to get yourself unstuck, but not more than two hours
```

Comment:
fly-by feedback: it may be helpful to clarify that/if this applies to non-engineers too and maybe signal some things that can help unblock them. 

I'm thinking mainly of people who are not technical and don't touch code regularly, like Lottie. If she's just staring at the code for an hour and not getting anywhere then that's arguably a waste of her time. If we're going to ask that she do that we can at least sign-post a few things to help. 

```suggestion
  - Whether you're an engineer or not, spend at least an hour on trying to get yourself unstuck, but not more than two hours
  - AI tools, such as ChatGPT, can be really helpful ways for less technical team members to get sections of code explained to them. Give them a go!
```